test/integration/query-interface/createTable.test.js

```diff
@@ -123,7 +124,7 @@ describe(Support.getTestDialectTeaser('QueryInterface'), () => {
         });
 
         const table = await this.queryInterface.describeTable('SomeTable');
-        if (dialect.includes('postgres')) {
+        if (dialect.includes('postgres') || dialect.includes('yugabytedb')) {
```

Comment:
the reason for `includes` is that it could be `postgres` or `postgres-native`, you should still use `===`

```suggestion
        if (dialect.includes('postgres') || dialect === 'yugabytedb') {
```